csfieldguide/templates/chapters/index.html

```diff
@@ -14,12 +14,11 @@ <h1 id="chapters" class="section-heading anchor-link">Chapters</h1>
 {% block content %}
 
   <div class="row chapter-list">
-
       {% if all_chapters %}
           {% for chapter in all_chapters %}
           <div class="l12">
             <div class="row">
-              <a href="{% url 'chapters:chapter' chapter.slug %}">
+              <a href="{% url 'chapters:chapter_section' chapter.slug chapter.first_section.slug %}">
```

Comment:
Pretty sure the original code here is correct, as a link from the chapter index will jump straight to the first section, missing the introduction!